src/output.rs

```diff
@@ -96,7 +96,7 @@ pub fn output_activedata_etl(results: CovResultIter, output_file: Option<&str>,
         for function in result.functions.values() {
             start_indexes.push(function.start);
         }
-        start_indexes.sort();
+        start_indexes.sort_unstable();
```

Comment:
I suppose that all `function.start` are different so I guess that using `sort_unstable` doesn't make any difference in the final order with `sort`. Out of curiosity, do you have a link on some benchmarks to compare perf for an array with unique elements ?
Anyway we should maybe use a `btreeset` here.

Reply:
I don't have any benchmarks for that, mostly just followed clippy's recommendations.
It probably doesn't make a difference in this case and just satisfies Google 😅